src/main/kotlin/eu/kevin/api/serializers/LocalDateSerializer.kt

```diff
@@ -18,6 +18,6 @@ object LocalDateSerializer : KSerializer<LocalDate> {
     }
 
     override fun deserialize(decoder: Decoder): LocalDate {
-        return LocalDate.parse(decoder.decodeString(), format)
+        return LocalDate.parse(decoder.decodeString().substringBefore('T'), format)
```

Comment:
I believe we should fallback from timestamp format to the date format if first parse fails, so I would wrap this inside of try/catch and fallback to yyyy-MM-dd

Reply:
Done! I tried it yesterday as an alternative.